app/controllers/calculators_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class CalculatorsController < ApplicationController
-  before_action :find_calculator, except: :index
+  before_action :find_calculator, except: [:index, :calculator]
```

Comment:
Maybe it's better to change like this
```suggestion
  before_action :find_calculator, only: [:show, :calculate]
```